pkg/plugin/run.go

```diff
@@ -130,11 +130,12 @@ func (e *AutoscaleEnforcer) handleAgentRequest(req api.AgentRequest) (*api.Plugi
 
 	node := pod.node
 
-	// Check whether the pod *will* migrate, then update its resources, and THEN start its
-	// migration, using the possibly-changed resources.
-	mustMigrate := e.updateMetricsAndCheckMustMigrate(pod, node, req.Metrics)
-	// Don't migrate if it's disabled
-	mustMigrate = mustMigrate && e.state.conf.migrationEnabled()
+	mustMigrate := pod.migrationState == nil &&
+		// Check whether the pod *will* migrate, then update its resources, and THEN start its
+		// migration, using the possibly-changed resources.
+		e.updateMetricsAndCheckMustMigrate(pod, node, req.Metrics) &&
```

Comment:
I was going to comment "it would be cool collect this into a method with a more descriptive name, but this method looks like it has side effects (based on the name) which almost raises more questions, and probably makes this more fragile (or fragile seeming). 

Anyway, I think it's fine,

Reply:
:shrug: